index.d.ts

```diff
@@ -45,11 +56,13 @@ declare class QuickLRU<KeyType, ValueType> implements Iterable<[KeyType, ValueTy
 	[Symbol.iterator](): IterableIterator<[KeyType, ValueType]>;
 
 	/**
-	Set an item.
+	Set an item. Returns the instance.
+
+	Individual expiration of an item can be specified with the `maxAge` option. If not specified, the global `maxAge` value will be used in case it is specified on the constructor, otherwise the item will never expire.
```

Comment:
```suggestion
	Individual expiration of an item can be specified with the `maxAge` option. If not specified, the global `maxAge` value will be used in case it is specified in the constructor, otherwise the item will never expire.
```